app/serializers/spree_signifyd/user_serializer.rb

```diff
@@ -40,6 +40,10 @@ def aggregateOrderDollars
       completed_orders.sum(:total).to_f
     end
 
+    def phone
+      object.orders.order("created_at DESC").first.try!(:ship_address).try!(:phone)
```

Comment:
We should be getting the phone number from the ship address for the current order.

`object.ship_address.phone`